kore/test/Test/Kore/Step/Function/Integration.hs

```diff
@@ -537,7 +539,7 @@ test_functionIntegration =
 test_Nat :: [TestTree]
 test_Nat =
     [ plus zero varN `matches` plus zero one  $ "plus(0, N) ~ plus(0, 1)"
-    , plus (succ varM) varN `notMatches` plus zero one  $ "plus(Succ(M), N) !~ plus(0, 1) "
+    , plus (succ varM) varN `unsolved` plus zero one  $ "plus(Succ(M), N) !~ plus(0, 1) "
```

Comment:
Although it wasn't grammatically correct, `notMatches` was almost a verb, so it made the code readable. Why is `unsolved` better?

Reply:
I changed the name because I got confused about halfway through this pull request about whether or not the matcher can ever return `\bottom`. It can't, and I changed the name to emphasize that we are checking for "no solution".